backend/Squirrel.SqlService/Squirrel.SqlService.BLL/Models/DTO/View/ViewDetailInfo.cs

```diff
@@ -0,0 +1,8 @@
+namespace Squirrel.SqlService.BLL.Models.DTO.View;
+
+public class ViewDetailInfo
```

Comment:
The view must inherit from BaseDbItem because ContentDifferenceService uses generic methods based on the BaseDbItem class.